modular_splurt/code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -120,3 +120,13 @@ GLOBAL_LIST_INIT(micro_bricks_recipes, list( \
 /obj/item/stack/sheet/micro_bricks/get_main_recipes()
 	. = ..()
 	. += GLOB.micro_bricks_recipes
+
+// New list for SPLURT-specific cloth sheet crafting
+GLOBAL_LIST_INIT(cloth_recipes_splurt, list ( \
+	new/datum/stack_recipe("handmade collar", /obj/item/clothing/neck/petcollar/handmade, 2) \
+	))
```

Comment:
Is it really necessary to make it a global list if it isn't used anywhere else in the code? I think you could just add its contents to `.` in the proc itself

Reply:
That's a much better idea. Thanks for the suggestion.